trailblazer/store/models.py

```diff
@@ -78,7 +78,7 @@ class Analysis(Model):
         types.Enum(*WorkflowManager.list()), default=WorkflowManager.SLURM.value
     )
 
-    failed_jobs = orm.relationship("Job", cascade="all,delete", backref="analysis")
+    jobs = orm.relationship("Job", cascade="all,delete", backref="analysis")
```

Comment:
does this have any backward incompatibility in the database?

Reply:
No it should not